### ISSUE TITLE
[Feature Request] Show Readme when no tabs are opened

### ISSUE THREAD

**Issue by vdegenne** [microsoft/vscode#174252]
Currently when no tabs are opened VSCode will show this screen:

![image](https://user-images.githubusercontent.com/2827383/218478429-39b35889-e15f-4b7f-98e6-d16dc0da122a.png)

It would be nice to have an option to display the processed markdown readme file instead if the project contains one.

**Comment by vdegenne** [comment]
@gjsjohnmurray not quite, this setting will only open the readme only once and the project window opens, plus it just opens the readme in a tab.  
What I meant was a way to always show the readme instead of the default page (with shortcuts as seen in the screenshot above) and not in the form of a tab, just the parsed readme with styles in the box.  
It could serve as a gateway during development, for instance we could write a list of todo so we are always reminded what to do next, if we were to write file urls with line descriptors we could quickly jump to a location in code where something needs to be done, or just to refresh the mind about the nature of a project architecture, etc... the application is endless because readme is user-defined.

Isn't it something worth considering?


By the way `"workbench.startupEditor": "readme"` will also mess up with ` "window.restoreWindows": "none"`, for some reasons it prevents the welcome page to open on VSCode startup when no project is opened at all.

**Comment by VSCodeTriageBot** [comment]
This feature request is now a candidate for our backlog. The community has 60 days to [upvote](https://github.com/microsoft/vscode/wiki/Issues-Triaging#up-voting-a-feature-request) the issue. If it receives 20 upvotes we will move it to our backlog. If not, we will close it. To learn more about how we handle feature requests, please see our [documentation](https://aka.ms/vscode-issue-lifecycle).

Happy Coding!

**Comment by VSCodeTriageBot** [comment]
This feature request has not yet received the 20 community [upvotes](https://github.com/microsoft/vscode/wiki/Issues-Triaging#up-voting-a-feature-request) it takes to make to our backlog. 10 days to go. To learn more about how we handle feature requests, please see our [documentation](https://aka.ms/vscode-issue-lifecycle).

Happy Coding!

**Comment by VSCodeTriageBot** [comment]
:slightly_frowning_face: In the last 60 days, this feature request has received less than 20 community upvotes and we closed it. Still a big Thank You to you for taking the time to create this issue! To learn more about how we handle feature requests, please see our [documentation](https://aka.ms/vscode-issue-lifecycle).

Happy Coding!
